main.py

```diff
@@ -194,8 +201,9 @@ def create_and_run_bot(parameters, llm_api_key):
 
 @click.command()
 @click.option('--resume', type=click.Path(exists=True, file_okay=True, dir_okay=False, path_type=Path), help="Path to the resume PDF file")
+@click.option('--browser', type=click.Choice(['chrome', 'firefox']), default=None, help='Browser to use for the bot')
 @click.option('--collect', is_flag=True, help="Only collects data job information into data.json file")
-def main(collect: False, resume: Path = None):
+def main(collect: False, browser: str = 'chrome', resume: Path = None):
```

Comment:
can you set existing chrome as default? Also make this configurable from config file

Reply:
Thanks for the suggestion. I’ve updated it so that it can now only be configured from the `config.yaml` file in the `data_folder`.

Reply:
ah this needs to be config.py. You can see that file in the release branch.

we renamed app_config.py to config.py

Reply:
fixed, thanks